app/presenters/hyrax/collection_presenter_decorator.rb

```diff
@@ -81,6 +81,16 @@ def banner_file
                        end
     end
 
+    def thumbnail_record
```

Comment:
its odd to me that this method is _record and not _file like the banner above. Also it is not memoized or have an exception catch like the one above.